cmake/DD4hepConfig.cmake.in

```diff
@@ -25,9 +25,17 @@ set ( DD4HEP_USE_CXX14   "@DD4HEP_USE_CXX14@" )
 set ( Geant4_DIR         "@Geant4_DIR@" )
 set ( GEANT4_USE_CLHEP   "@GEANT4_USE_CLHEP@" )
 
-set ( ROOTSYS            "$ENV{ROOTSYS}" )
-set ( ROOT_DIR           "$ENV{ROOTSYS}/cmake" )
-set ( ROOT_VERSION       "@ROOT_VERSION@" )
+
+if(NOT ENV{ROOTSYS})
```

Comment:
What does this have to do with the `share` directory? And why set ROOTSYS etc. to DD4hep_DIR?